turbo.json

```diff
@@ -1,32 +1,27 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "globalDependencies": ["**/.env.*local"],
-  "globalEnv": ["RUNNER_OS"],
-  "pipeline": {
+  "ui": "tui",
+  "globalDependencies": [".npmrc", ".prettierrc.cjs", "**/.env.*local"],
+  "tasks": {
     "build": {
       "env": ["GITHUB_ACTIONS"],
       "outputs": [".svelte-kit/**", "dist/**", "dist-svelte/**"],
-      "dependsOn": ["^build"],
-      "outputMode": "new-only"
+      "dependsOn": ["@sanity/insert-menu#build", "^build"]
     },
     "test": {
       "env": ["GITHUB_ACTIONS"],
       "dependsOn": ["^build"],
       "cache": false
     },
     "lint": {
-      "outputMode": "errors-only"
+      "outputLogs": "errors-only"
     },
     "preview": {
       "dependsOn": ["^build"]
     },
     "dev": {
-      "dependsOn": ["^build"],
```

Comment:
This will be added back once `@sanity/pkg-utils` can run a watcher that also handles `d.ts` generation